chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/DeveloperManagerTest.java

```diff
@@ -237,4 +238,19 @@ public void testDeveloperStatusChangeNotAllowedByNonAdmin()
 		
 		SecurityContextHolder.getContext().setAuthentication(null);
 	}
+	
+	/**
+	 * Given that I am authenticated to CHPL as any user
+	 * When I call the REST API's /decertified/developers, the controller calls the developerManager.getDecertifiedDevelopers()
+	 * Then the manager returns a list of DeveloperDecertifiedDTO with expected results
+	 */
+	@Transactional
+	@Rollback(true) 
+	@Test
+	public void testGetDecertifiedDevelopers() {
+		SecurityContextHolder.getContext().setAuthentication(adminUser);

```

Comment:
Doesn't this line suggest you're an authenticated user? But the "Given" above says "any user". Why authenticate? There's no security on this /decertifications call, right?